kubernetes/README.md

```diff
@@ -0,0 +1,17 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This section of the examples repository will contain a variety of uses cases for deployment of Confluent Platform and supporting applications deployed via [Confluent Operator](https://docs.confluent.io/current/installation/operator/index.html).
+
+# Use Cases
+| Demo | Info |
+|---|---|
+| [gke-base](https://docs.confluent.io/current/tutorials/kubernetes/gke-base/docs/index.html) | A base deployment of Confluent Platform on Google Kubernetes Engine (GKE) with sample data generation using [Kafka Connect](https://docs.confluent.io/current/connect/index.html)|
```

Comment:
Suggestion: remove link to https://docs.confluent.io/current/connect/index.html.

Reason: avoid user confusion on how to run the demo.  If a user is looking for a link on how to run it, they may miss the `gke-base` link and click on this link instead, which doesn't help them to run the demo.